### PR TITLE
Adapt-000 Remove animation variants but add translate variants

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Animation variants in TW aren't that useful for us, but we need the translate variants for CTA link icon animation. So removing the animation variants but adding the ones for `translate`.
- Add a few spacing steps

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Look at code

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: src/plugins/theme/spacing.js [hunk 1]
```diff
@@ -27,7 +27,10 @@ module.exports = function () {
     11: '1.1rem',
     12: '1.2rem',
     13: '1.3rem',
+    14: '1.4rem',
     15: '1.5rem',
+    16: '1.6rem',
+    17: '1.7rem',
```
Comment (yvonnetangsu) [inline src/plugins/theme/spacing.js:33]: Might as well adding these

File: tailwind.config.js [hunk 2]
```diff
@@ -97,8 +97,8 @@ module.exports = {
   // core utility plugin.
   // https://tailwindcss.com/docs/configuration/#variants
   variants: {
-    animation: ['hover', 'focus', 'hocus', 'group-hover', 'group-focus', 'group-hocus', 'motion-safe', 'motion-reduce'],
     extend: {
+      translate: ['hocus', 'group-hocus', 'motion-safe', 'motion-reduce'],
```
Comment (yvonnetangsu) [inline tailwind.config.js:101]: For CTA link icon animation

Comment (yvonnetangsu) [reply tailwind.config.js]: These have been changed - I'm pulling them in my Decanter-React branch to test

File: src/plugins/theme/spacing.js (hunk 1 above)
Comment (sherakama) [reply src/plugins/theme/spacing.js:33]: SG2M

**Review by sherakama — APPROVED** [review]
GTG

**Pull request merged**